rust/main/chains/hyperlane-aleo/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+//! Implementation of hyperlane for aleo.
+
+#![warn(missing_docs)]
+#![deny(clippy::unwrap_used, clippy::panic)]
+#![deny(clippy::arithmetic_side_effects)]
```

Comment:
I wonder if it is temporary and will go away with implementation?

Reply:
Let's check if we can change the code and remove some of these from Aleo. I don't mind if we add it back if the fix requires a lot of effort.